src/components/presentational/ProgressBar/ProgressBar.css

```diff
@@ -0,0 +1,50 @@
+.progress-bar {
+	display:inline-block;
+	margin: 10px 0px;
+	width: 100%;
+	max-width: 500px;
+	--icon_size: 24px;
+	--icon_border_width: 2px;
+}
+
+.progress-bar-fill {
+    position: absolute;
+	min-height: 1px;
+    height: calc( var(--icon_size) + 4px);
+	top:0;
+	border-radius: var(--icon_size);
+	z-index:1
+}
+
+.progress-bar .progress-bar-background {
+    border-radius: 500px;
+	border-width: var(--icon_border_width) ;
+	border-style: none;
+    background: #fff;
+	overflow: hidden;
+	width: 100%;
+	position: relative;
+	height:calc( var(--icon_size) + 4px);
+	z-index:1;
+}
+
+.progress-bar .progress-steps {
+	top:0;
+	margin: 10px 0px;
+	width: 100%;
+	max-width: 500px;
+	position: absolute;
+	z-index: 2;
+	display:inline-block;
+}
+
+.progress-bar .step-icon {
+	position: absolute;
+	transform: translate(-100%, 0)
+}
+
+.progress-bar .step-icon span {
```

Comment:
I wouldn't assume a span will be used as the child element...

So there is actually some cool stuff you did in the story here for the various little star icons and how they are colored etc.

I would not just waste that in a story but would actually invest in a separate component called "ProgressBarStep"...it could be exported under the progress bar...such that we have some out of box progress bar steps that can be placed on this (but someone could also come up with their own to supply to this).

and the props could be like:
- borderColor?: ColorDefinition
- backgroundColor: ColorDefinition
- children  (this is where you'd put in the font awesome icon for the star, or the text 75 or whatnot)
- height (based on this it would auto-calculate the top margin to center itself vertically, e.g. for the large star)


e.g. for the big star at the end your step would be like:

```
steps: [{
		stepPercent: 100,
		stepIcon: <ProgressBarStep borderColor="yellow" backgroundColor="orange" height="24px">
<FontAwesomeIcon icon={faStar} style={{ color: "yellow" }} />
</ProgressBarStep>
	}],
```

Basically to have to limit the amount of work someone has to do to get it to look like the storybook, while still allowing some flexibility